web/elm/src/Dashboard/Grid/Constants.elm

```diff
@@ -21,17 +21,27 @@ cardBodyHeight =
 
 padding : number
 padding =
-    25
+    32
+
+
+carHeaderVerticalSpace : number
+carHeaderVerticalSpace =
+    8
 
 
 cardHeaderHeight : number -> number
 cardHeaderHeight numRows =
-    20 + 30 * numRows
+    32 + 21 * numRows + (numRows - 1) * carHeaderVerticalSpace
+
+
+groupHeaderLineHeight : number
+groupHeaderLineHeight =
+    25
 
 
 groupHeaderHeight : number
 groupHeaderHeight =
-    60
+    padding + groupHeaderLineHeight
```

Comment:
Now that the padding is included in the `groupHeaderHeight` (which makes sense in general, I think), it's causing issues with a previous computation for lazy rendering. We need to get rid of

https://github.com/concourse/concourse/blob/5530bdc6ee6461af68d6d98cf73b9c2f8f723b71/web/elm/src/Dashboard/Dashboard.elm#L1470

...or when there are many teams, pipelines will start to disappear:


https://user-images.githubusercontent.com/26583442/108868309-1453ad00-75c4-11eb-999e-c69abfce6604.mov